### PR TITLE
Update IS_COMMAND definitions to use MOD_MASK_SHIFT

### PR DESCRIPTION
**Pull request by vomindoraan**
## Description

Update the `IS_COMMAND` default definition, as well as examples in docs and `config.h` templates, to use the `MOD_MASK_SHIFT` constant introduced in #4337.

`MOD_MASK_SHIFT` is equivalent to the current, manual definition, but it's shorter and easier to read:

https://github.com/qmk/qmk_firmware/blob/bc801a6e38fce8d5b5fad6edecbbdffe37651b70/tmk_core/common/keycode.h#L50

## Types of Changes

- [x] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by vomindoraan** [comment]
CI errors seem unrelated to this PR (I believe they're fixed by #6343).

**Comment by vomindoraan** [comment]
CI is passing now.

File: docs/feature_command.md [hunk 1]
```diff
@@ -16,36 +16,36 @@ To use Command, hold down the key combination defined by the `IS_COMMAND()` macr
...
-|`MAGIC_KEY_EEPROM`                  |`E`                                                                        |Print stored EEPROM config to the console       |
-|`MAGIC_KEY_EEPROM_CLEAR`            |`BSPACE`                                                                   |Clear the EEPROM                                |
-|`MAGIC_KEY_NKRO`                    |`N`                                                                        |Toggle N-Key Rollover (NKRO)                    |
-|`MAGIC_KEY_SLEEP_LED`               |`Z`                                                                        |Toggle LED when computer is sleeping            |
+|Define                              |Default                                    |Description                                     |
+|------------------------------------|-------------------------------------------|------------------------------------------------|
+|`IS_COMMAND()`                      |<code>(get_mods() == MOD_MASK_SHIFT)</code>|The key combination to activate Command         |
```
Comment (fauxpark) [inline docs/feature_command.md]:
```suggestion
|`IS_COMMAND()`                      |`(get_mods() == MOD_MASK_SHIFT)`|The key combination to activate Command         |
```

Comment (vomindoraan) [reply docs/feature_command.md]: Fixed, thanks!

**Pull request merged**